### ISSUE TITLE
Inconsistent date format on JSON return object

### ISSUE THREAD

**Issue by GoodMorgon** [Swechhya/excelR#56]
Using both the dev and latest cran version and editing a calendar type column I receive the error `character string is not in a standard unambigous format`. 

Stepping into the code and printing the JSON object without using `excel_to_R` I can see that the calendar input is of the format `YYYY-MM-DD 00:00:00` instead of just `YYYY-MM-DD`. 

However, I cannot reproduce this issue by using the example below. 

```
   library(shiny)                                                                        
      library(excelR)                                                                       
                                                                                            
       shinyApp(                                                                            
         ui = fluidPage(excelOutput("table")),                                              
         server = function(input, output, session) {                                        
          iris <- iris
          iris$test <- as.Date("2019-01-01")

           output$table <- renderExcel(excelTable(data = head(iris)))                                        
          observeEvent(input$table,{                                                        
            print(excel_to_R(input$table))                                                  
          })                                                                                
          }                                                                                 
        )
```

Please refer to the attached print-screens for the JSON output and associated `excelTable` changed table cell.
![JSON output](https://user-images.githubusercontent.com/55554176/70050470-4f3c8000-15cf-11ea-9c17-608d503034db.png)
![Table Value](https://user-images.githubusercontent.com/55554176/70050476-519eda00-15cf-11ea-8571-833e7e0a146a.png)

**Comment by marciz** [comment]
I think the issue is blank in the date field
> as.Date("")
Error in charToDate(x) : 
  character string is not in a standard unambiguous format

**Comment by GoodMorgon** [comment]
That is probably another issue. You can see from the screenshots that the date input indeed is passed along, but with a timestamp addon.

**Comment by Swechhya** [comment]
@marciz This issue cannot be replicated even with an empty cell with the example below:
```
library(shiny)                                                                        
library(excelR)                                                                       

shinyApp(                                                                            
    ui = fluidPage(excelOutput("table")),                                              
    server = function(input, output, session) {                                        
        iris <- iris
        iris$test <- c(rep(as.Date("2019-01-01"),5,),"")
        
        output$table <- renderExcel(excelTable(data = head(iris)))                                        
        observeEvent(input$table,{                                                        
            print(excel_to_R(input$table))                                                  
        })                                                                                
    }                                                                                 
)
```
Can you please post an example which replicates this issue so that I can merge your PR?

**Comment by marciz** [comment]
@Swechhya , try to delete test row 1. It gets an error

**Comment by Swechhya** [comment]
@marciz Using the latest development version and deleting the first row I am not able to replicate the error. Any pointers why?

**Comment by marciz** [comment]
Press delete button on the first cell of test column. Is there still no error?

**Comment by Swechhya** [comment]
@marciz Can replicate it. Thank you 😃

**Comment by Swechhya** [comment]
@GoodMorgon  Closing this issue as this is fixed by PR #60.  Thank you @marciz for the fix 🎉
